src/components/projectcard.jsx

```diff
@@ -2,33 +2,29 @@
 
 import React from "react";
 import Image from "next/image";
+import temp from "public/tempimage.png";
 
 const ProjectCard = ({ project }) => {
-  const { projectName, icon, year, names, description } = project;
+  const { projectName, year, names, description } = project;
 
   return (
-    <div className="flex-col justify-self-center justify-center items-center shadow-2xl  min-w-96 h-full w-96  rounded-md md:w-5/6 lg:w-7/12   sm:w-5/6 mt-3">
+    <div className="flex h-full w-11/12 flex-col justify-center items-center shadow-lg rounded-xl m-0 md:m-2 mt-3">
```

Comment:
add overflow-hidden to hide the corners
<img width="414" alt="Screenshot 2024-03-28 at 9 14 22 PM" src="https://github.com/acm-ucr/senior-design-website/assets/55073341/af87b247-e4f4-4144-9681-c46edc206009">